pkg/traces/exporter/client_authenticator.go

```diff
@@ -7,12 +7,14 @@ import (
 
 // Implements google.golang.org/grpc/credentials.PerRPCCredentials interface
 type clientAuthenticator struct {
-	token string
+	token      string
+	disableTLS bool
 }
 
-func newClientAuthenticator(token string) *clientAuthenticator {
+func newClientAuthenticator(token string, disableTLS bool) *clientAuthenticator {
```

Comment:
With `setDisableTLS` do you need to change the signature here? It feels weird to change it for something only in CI